tw-tkms-starter/build.gradle

```diff
@@ -62,6 +62,7 @@ dependencies {
 
     testRuntimeOnly libraries.mariadbJavaClient
     testRuntimeOnly libraries.postgresql
+    testRuntimeOnly libraries.javaxValidationApi
```

Comment:
Maybe let's add a comment here why this is added so that we'd know when to clean it up.